src/functionalTest/groovy/ReportsFunctionalTest.groovy

```diff
@@ -70,4 +70,72 @@ class ReportsFunctionalTest extends Specification {
         new File("$projectDirRealPath/build/reports/tests-execution/json/test.json").exists()
         new File("$projectDirRealPath/build/reports/tests-execution/html/test.html").exists()
     }
+
+    def "should replace minority sign and colon characters in test names"() {
```

Comment:
How about `should replace special characters in test names`?

Reply:
Renamed